### PR TITLE
Release preparation

### PR DESCRIPTION
**Pull request by jakobbraun**

### CODE REVIEW

File: doc/changes/changelog.md [hunk 1]
```diff
@@ -0,0 +1 @@
+changes_0.3.0.md
```
Comment (AnastasiiaSergienko) [inline doc/changes/changelog.md:1]: Let's make this file unified: 
https://raw.githubusercontent.com/exasol/sql-statement-builder/master/doc/changes/changelog.md

File: doc/changes/changes_0.3.0.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# dynamodb-virtual-schema 0.3.0, released 2020-07-08
+ 
+## Summary
+This pre-release introduces selection & projection pushdown for DynamoDB query generation
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_0.3.0.md:4]: One empty line after the header

File: doc/changes/changes_0.3.0.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# dynamodb-virtual-schema 0.3.0, released 2020-07-08
+ 
+## Summary
+This pre-release introduces selection & projection pushdown for DynamoDB query generation 
+and query parallelization. 
+
+## Features / Enhancements
+ 
+* Selection push-down to for DynamoDB requests
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_0.3.0.md:9]: Do you have issue numbers?

File: doc/changes/changelog.md (hunk 1 above)
Comment (jakobbraun) [reply doc/changes/changelog.md:1]: ah, ok. There I missunderstood the template and thought it should be a symlink.

> Link from doc/changes/changelog.md to new log entry file

By the way, in the template it says, we should use changes_ with an underscore. In the link above, you used and `-`

File: doc/changes/changes_0.3.0.md (hunk 3 above)
Comment (jakobbraun) [reply doc/changes/changes_0.3.0.md]: yes. I added them now.

In itelij Idea it looks however not so nice:
grafik

Comment (AnastasiiaSergienko) [reply doc/changes/changes_0.3.0.md]: No, github displays it in a normal way. It's a usual way to reference a ticket

**Pull request merged**